src/components/widgets/InputPopup.ts

```diff
@@ -194,11 +194,6 @@ export class InputPopup extends EventEmitter {
             return
         } // Continue only if the result is 0
         const v = this.value
-        if (v.toString().length < 20 && key.sequence.length === 1) {
```

Comment:
Why "<20"? it should not have a size limit. or if yes, this limit should be parametrized in the props. ;)

Reply:
Not quite sure, that was copied from the original code. 197-201 has just been moved to  241-245 to the switch/case takes priority

Reply:
Original code in main: https://github.com/Elius94/console-gui-tools/blob/5fceb67daf2b6daff872b967a5aa8936bdcbfe0a/src/components/widgets/InputPopup.ts#L121

Reply:
ah yes sorry, it's about the numeric!
I forgot I put that fixed limit myself, to avoid entering more than 20 digits.